### PR TITLE
Update README.md with details of talks and code of conduce

### PR DESCRIPTION
**Pull request by ofchurches**
Hi Jonathan,

I've added in stuff for the sep 25 meet up. More to come. Could you review this and merge it if its ok with you?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,26 @@
...
+Putting R in the internet tubes; An introduction to blogdown! Blogdown is an R package that lets you create websites from R Markdown documents, making it easy to put your R code and outputs online. It creates a static website you can host on any server with minimal effort. In this talk we'll cover what blogdown is and why you might want to use it, look at some easy ways to get started, and share resources to take you further. 
+
+###Rachel Searston
+
+
+###Rory Tarnow-Mordi
+Sometimes our datasets are just too darn big. In this talk we'll look at using SparkR to spread our data and data transformations over a cluster of machines.
+
+##Code of conduct
+All participants at this meet-up agree to abide by the R Consortum [code of conduct](https://wiki.r-consortium.org/view/R_Consortium_and_the_R_Community_Code_of_Conduct). In particular, as members of the R comunnity, participants agree to be:
```
Comment (jonocarroll) [inline README.md]: \* Consortium

File: README.md [hunk 2]
```diff
@@ -1,2 +1,26 @@
-# GeneralDiscussion
-Ideas, Thoughts, Suggestions relevant to the Adelaide R Users Group Meetup
+#Spring meet-up
```
Comment (jonocarroll) [inline README.md:1]: \* 2019

**Review by jonocarroll — APPROVED** [review]
Looks good. I'm on mobile right now but if you can fix these minor points I'll merge ASAP. Cheers!

**Comment by ofchurches** [comment]
Hi @jonocarroll, sorry I'm still finding my way around GitHub. I've changed "Consortum' to "Consortium" in the fork in my repo and committed the changes to master. It looks like its come through here automatically. Is there anything else I should do?

**Comment by jonocarroll** [comment]
I'll add the 2019 to the section title. Cheers for the work!

**Comment by ofchurches** [comment]
Ah, sorry. I missed that. Thanks Jonathan.

Best, Owen

**Pull request merged**